### PR TITLE
[aclshow]: Add aclshow utility to view ACL info and counters

### PR DESCRIPTION
**Pull request by andriymoroz-mlnx**

### CODE REVIEW

File: scripts/aclshow [hunk 1]
```diff
@@ -0,0 +1,326 @@
...
+
+        if tables:
+            self.table_list = tables.split(",")
+
+        self.setup_db_connections()
+        self.read_port_map()
+        self.validate_ports()
+
+    def setup_db_connections(self):
+
```
Comment (jleveque) [inline scripts/aclshow]: Remove empty line for consistency.

File: scripts/aclshow [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+                    table_port_list = table_props['ports'].split(",")
+                    if not self.intersect(self.port_list, table_port_list):
+                        continue
+                self.acl_tables[table_name] = table_props
+
+            if verboseflag:
+                print("ACL Tables to show:", len(self.acl_tables))
+
+        def fetch_acl_rules():
+
```
Comment (jleveque) [inline scripts/aclshow]: Remove empty line for consistency

**Comment by andriymoroz-mlnx** [comment]
Updated. Now we need sswsdk on switch

**Comment by liatgrozovik** [comment]
Can we merge this tool and update submodule pointers?

**Comment by lguohan** [comment]
@stcheng and @jleveque , please approve or indicate if there is any further changes needed?

File: scripts/aclshow [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+        self.table_list = []
+
+        if rules:
+            self.rule_list = rules.split(",")
+
+        if tables:
+            self.table_list = tables.split(",")
+
+        # Set up db connections
+        self.db = sswsdk.SonicV2Connector()
```
Comment (jleveque) [inline scripts/aclshow]: Should change from `sswsdk.SonicV2Connector()` to `swsssdk.SonicV2Connector()`

Comment (andriymoroz-mlnx) [inline scripts/aclshow]: Fixed

**Pull request merged**